nitrous.readme.md

```diff
@@ -0,0 +1,68 @@
+# Nitrous Brunch Quickstart
```

Comment:
Does this belong to the core repo? If anything, could we create a separate repo, perhaps something like `brunch-nitrous` that will contain this doc and the scripts? In which case, they can be at the root, not in a subdir

As not to pollute the main repo... What do you think?